apps/question-bank/src/test/java/gov/cdc/nbs/questionbank/page/content/staticelement/HyperLinkSteps.java

```diff
@@ -19,16 +23,26 @@ public class HyperLinkSteps {
     @Autowired
     private StaticRequest request;
 
+    @Autowired
+    private ObjectMapper mapper;
+
     @Autowired
     private PageMother mother;
 
     @Autowired
     private ExceptionHolder exceptionHolder;
 
     private final Active<ResultActions> response = new Active<>();
+    private final Active<ResultActions> updateResponse = new Active<>();
```

Comment:
suggestion (non-blocker):  if the `Active<ResultActions>` is reset it can be reused for add and update as they are both `ResultActions` and are called at different times.